frodo.js

```diff
@@ -1,32 +1,30 @@
 /**
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?